master/lustrefactory.py

```diff
@@ -56,6 +62,65 @@ def buildzfsCommand(props):
     return args
 
 @util.renderer
+def configureCmd(props):
+    args = ["./configure"]
+    style = props.getProperty('buildstyle')
+
+    if style == "deb" or style == "rpm":
```

Comment:
If we expect a proliferation of conditionals, then I agree it makes sense to break it out by platform. If this code will remain relatively static, I'd keep it consolidated. Without more context I can't say which is the case.